src/com/lftechnology/java/training/srijan/basics/coreapp/ui/EmployeeView.java

```diff
@@ -0,0 +1,397 @@
+	private int userStatus;
+	private String username;
+	private String fullname;
+	private String department;
+	private String address;
+	private String password;
+
+	public EmployeeView() {
+		this.choiceUserRole = 0;
+		this.userStatus = 0;
+		this.username = "";
+		this.fullname = "";
+		this.department = "";
+		this.address = "";
+		this.password = "";
+	}
+
+	/**
+	 * <p>Takes employee information from user</p>
+	 * 
+	 * @param e
+	 * @return
+	 * @author srijan
+	 */
+	public Employee inputUserInfo(Employee e, Scanner input) {
+
+		Boolean checkUserValidity;
+		try {
+			input = new Scanner(System.in);
+			do {
+				LOGGER.log(Level.INFO, "Enter Username*:");
+				this.username = input.nextLine();
+				e.setUserName(username);
+				checkUserValidity = employeeControl.checkUsernameValidation(e);
+				if (checkUserValidity.equals(true)) {
+					LOGGER.log(
+						Level.INFO,
+						"Duplicate Username or Empty Username..Please choose another Username");
+				}
+				else {
+					LOGGER.log(Level.INFO, "valid Username");
+				}
+			}
+			while (checkUserValidity.equals(true));
+			LOGGER.log(Level.INFO, "Enter Fullname*:");
+			this.fullname = input.nextLine();
+			e.setFullName(fullname);
+			LOGGER.log(Level.INFO, "Enter Department*:");
+			this.department = input.nextLine();
+			e.setDepartment(department);
+			LOGGER.log(Level.INFO, "Enter Address:");
+			this.address = input.nextLine();
+			e.setAddress(address);
+			LOGGER.log(Level.INFO, "Enter password*");
+			this.password = input.nextLine();
+			e.setPassword(password);
+			LOGGER.log(
+				Level.INFO,
+				"Enter User Role*:\n1.ADMIN\n2.NORMAL\n3.Default: NORMAL");
+			this.choiceUserRole = input.nextInt();
+			switch (choiceUserRole) {
+			case 1:
+				e.setUserRole(UserRole.ADMIN);
+				break;
+			case 2:
+				e.setUserRole(UserRole.NORMAL);
+				break;
+			default:
+				LOGGER.log(
+					Level.INFO,
+					"please choose right option,Default Normal user assigned");
+				e.setUserRole(UserRole.NORMAL);
+				break;
+			}
+			LOGGER.log(
+				Level.INFO,
+				"User status:\n0.Not terminated\n1.Terminated\n2.Default: Not Terminated");
+			userStatus = input.nextInt();
+			switch (userStatus) {
+			case 0:
+				e.setIsTerminated(false);
+				break;
+			case 1:
+				e.setIsTerminated(true);
+				break;
+			default:
+				LOGGER.log(
+					Level.INFO,
+					"Wrong option, User has been default assigned to Not Terminated");
+				e.setIsTerminated(false);
+				break;
+			}
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception", ex);
+		}
+		return e;
+	}
+
+	/**
+	 * <p>Calls different user methods depending upon user choice</p>
+	 * 
+	 * @param normalUserChoice
+	 * @param userName
+	 * @author srijan
+	 */
+	public void normalUserOptionHandler(int normalUserChoice, Employee em,Scanner input) {
+
+		switch (normalUserChoice) {
+		case 1:
+			employees = employeeControl.viewAllUser(em);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		case 2:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 9 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1151)